### PR TITLE
gnome-keyring: update to 46.1.

### PR DESCRIPTION
**Pull request by fanyx**
#### Testing the changes
- I tested the changes in this PR: **NO**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by fanyx** [comment]
@ericonr 
can you test these changes?
i disable some features of gnome-keyring on my system so only part this is verified.

File: srcpkgs/gnome-keyring/template [hunk 1]
```diff
@@ -14,7 +14,7 @@ license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/GnomeKeyring/"
 changelog="https://gitlab.gnome.org/GNOME/gnome-keyring/-/raw/master/NEWS"
 distfiles="${GNOME_SITE}/$pkgname/${version%.*}/$pkgname-$version.tar.xz"
```
Comment (oreo639) [inline srcpkgs/gnome-keyring/template:16]: Replace $pkgname here with `gnome-keyring`, and $version with `${version}`

**Comment by oreo639** [comment]
The ssh agent was moved from gnome-keyring to gcr4 (which we don't currently ship), so I will look in to that later: https://gitlab.gnome.org/GNOME/gnome-keyring/-/issues/140

**Comment by oreo639** [comment]
Lets just build it with `--enable-ssh-agent` for now as that is what other distros are doing.

**Pull request merged**